hydra-plutus/hydra-plutus.cabal

```diff
@@ -137,7 +137,7 @@ test-suite tests
     , QuickCheck
     , time
 
-  build-tool-depends: hspec-discover:hspec-discover -any
```

Comment:
what is the benefit we gain by removing this `-any` flag?

Reply:
I guess then at compilation time cabal will pick exact version of hspec-discover it has in scope. This is something we did in ensamble I think, can't really remember but I do prefer that we don't pick any random version that works of hspec-discover any way.